examples/light-js/index.html

```diff
@@ -88,7 +89,8 @@
           return;
         }
         statusDiv.innerHTML = "<p>Dialing peer.</p>";
-        await node.dial(ma, ["filter", "lightpush"]);
+        const multiaddr = MultiformatsMultiaddr.multiaddr(ma);
```

Comment:
Gonna address it by this comment:
https://github.com/waku-org/js-waku-examples/pull/225/files#r1164803477